projects/elfutils/Dockerfile

```diff
@@ -17,6 +17,11 @@
 FROM gcr.io/oss-fuzz-base/base-builder
 RUN apt-get update && \
     apt-get install -y pkg-config make autoconf autopoint zlib1g-dev flex bison gawk
-RUN git clone --depth 1 git://sourceware.org/git/elfutils.git
+RUN bash -x -c "\
+if [[ '$CIFUZZ' == True ]]; then \
```

Comment:
FWIW I'm not sure why the skia project started to use `true` (instead of `True`) in https://github.com/google/oss-fuzz/pull/4588/files. As far as I can see the variable is always set to `True` on CIFuzz